openstacksdk_plugin/resources/compute/server.py

```diff
@@ -32,6 +32,74 @@
                                            SERVER_TASK_STOP,)
 
 
+def _set_server_ips_runtime_properties(server):
+    addresses = server.addresses
+    if not addresses:
+        return None
+
+    ipv4_addresses = []
+    ipv6_addresses = []
+
+    for network, addresses in addresses.iteritems():
+        for address in addresses:
+            # ip config
+            ipv4 = dict()
+            ipv4['addr'] = address['addr']
+            ipv4['type'] = address['OS-EXT-IPS:type']
+
+            # Check where `ip_config` should be added
+            if address['version'] == 4:
+                ipv4_addresses.append(ipv4)
+            elif address['version'] == 6:
+                ipv6_addresses.append(address['addr'])
+
+    # Check if access_ipv4 is set or not
+    if server.access_ipv4:
+        ctx.instance.runtime_properties['access_ipv4'] = server.access_ipv4
+
+    # Check if access_ipv6 is set or not
+    if server.access_ipv6:
+        ctx.instance.runtime_properties['access_ipv6'] = server.access_ipv6
+
+    # If "ipv4_addresses" is only contains one item, them we need to check
+    # both private/public ip in order to set them as part of runtime_properties
+    for ipv4 in ipv4_addresses:
+        ip = ipv4['addr']
+
```

Comment:
How do we know we are getting the primary interface?

Reply:
Actually We do not know :( because the response returned does not specify which one is the primary one

Reply:
@EarthmanT Why should we add it ? it is not part of the node and we can get the floating ip  when the server is created

Reply:
Do the addresses object say which device name, e.g. `eth0` is set?

Reply:
No, the only info contains within each address are the following :
```
{u'OS-EXT-IPS-MAC:mac_addr': u'fa:16:3e:da:44:48', u'version': 4, u'addr': u'10.0.0.36', u'OS-EXT-IPS:type': u'fixed'}
```